### PR TITLE
Add ant animation

### PR DESCRIPTION
**Pull request by duthades**
This PR changes folowing 

 - Add ant animation
 - Made a BaseSprite class
 - Added a asset folder
 - Deleted old ant image

### CODE REVIEW

File: src/base_sprite.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import os
+
+
+class BaseSprite:
+    def update(self):
+        raise NotImplementedError("update method must be implemented in sub-classes")
+
+    def load_animation(self, duration=0.1):
+        images = []
+        pyglet.resource.path = [self._image_path]
```
Comment (duthades) [inline src/base_sprite.py]: Pass image path as a method argument.

Comment (duthades) [reply src/base_sprite.py]: removed base_sprite

**Review by VismayVakharia — APPROVED** [review]
Looks good!

**Review by aditya95sriram — APPROVED** [review]
Acceptable

**Pull request merged**